src/components/TeamWorkspace/TeamWorkspace.tsx

```
@@ -0,0 +1,861 @@
+}
+
+interface Activity {
+  id: string;
+  user: TeamMember;
+  action: string;
+  target: string;
+  timestamp: string;
+  type: 'create' | 'update' | 'delete' | 'comment' | 'share';
+}
+
+interface Comment {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Comment'. Consider to delete it or rename it.

'Comment' is defined here:


(lint/suspicious/noRedeclare)